packages/sit-onyx/src/components/OnyxColorSchemeMenuItem/OnyxColorSchemeMenuItem.stories.ts

```diff
@@ -0,0 +1,30 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxColorSchemeMenuItem from "./OnyxColorSchemeMenuItem.vue";
+
+/**
+ * Pre-built menu item for the `OnyxUserMenu` that can be used inside the nav bar to
+ * display the current color scheme to the user and allow changing it by displaying a `OnyxColorSchemeDialog`.
+ */
+const meta: Meta<typeof OnyxColorSchemeMenuItem> = {
+  title: "Navigation/modules/ColorSchemeMenuItem",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxColorSchemeMenuItem,
+    events: ["update:modelValue"],
+    decorators: [
+      (story) => ({
+        components: { story },
+        template: `<div style="max-width: 16rem;"> <story /> </div>`,
+      }),
+    ],
+  }),
+};
+
```

Comment:
do you get this warning in the console, too? (probably out of scope of this PR, but if there is a quick fix...)

![image](https://github.com/SchwarzIT/onyx/assets/151019977/01068f31-f9d0-448f-8231-77937c1650ea)

Reply:
So I'd say this is only a Storybook related issue

Reply:
aah makes sense, thank you!